lib/v2/baozimh/chapters.js

```diff
@@ -0,0 +1,90 @@
+const got = require('@/utils/got');
```

Comment:
Please fix `lib/v2/baozimh/index.js` instead of reinventing the wheel as it should parse all latest chapters' pages.

Reply:
done

Reply:
Please take a look at this issue again

Reply:
@TonyRL  I have to do this because the response code may not be 200, and if I use a wrapped got, then I will never get the data in the future

Reply:
> Please fix `lib/v2/baozimh/index.js` instead of reinventing the wheel as it should parse all latest chapters' pages.

I'm talking about this whole `lib/v2/baozimh/chapters.js` file